packages/application-shell/src/components/login-sso-callback/login-sso-callback.spec.js

```diff
@@ -64,7 +64,7 @@ describe('lifecylcle', () => {
       });
       it('should call requestAccessToken with idToken', () => {
         expect(props.requestAccessToken).toHaveBeenCalledWith(
-          expect.objectContaining({ idToken: '111' })
+          expect.objectContaining({ idToken: 'abcTestIdToken' })
```

Comment:
Cause 111 will be casted to an int as qss casts boolean and numbers for us I felt this may be a better test token name.